app/controllers/subscriptions.js

```diff
@@ -184,7 +184,9 @@ export default Ember.Controller.extend(Ember.Evented, {
     var existingItem = this.store.peekRecord(type, item.id);
 
     // update_store message is sent before response to APP save so ignore
-    var fromCurrentUser = parseInt(data.sender.user.id, 10) === parseInt(this.session.get("currentUser.id"), 10);
+    if (data.sender.user){
```

Comment:
@patrixr we had an issue on staging whereby if `data: {sender: null}` we throw an error which causes socketio to endlessly re-send the same data. We'll address `sender` on API but also need to program defensively in this case to prevent thousands of errors (literally!). How does this look?